src/components/GameItem/index.tsx

```diff
@@ -1,72 +1,44 @@
 import { GameItemType } from '../../App';
-import StarIcon from '@mui/icons-material/Star';
-import styled from '@emotion/styled';
+import { Card, CardMedia, CardContent, Typography, Box } from '@mui/material';
+import { makeStyles } from '@mui/styles';
 
-const GameItemContainer = styled.div`
-  background: #ececec;
-  padding: 16px 32px;
-  display: flex;
-  gap: 16px;
-  border-radius: 4px;
-  box-shadow: 0 0 6px rgba(0, 0, 0, 0.08);
-`;
+const useStyles: any = makeStyles(() => ({
+  card: { display: 'flex', width: '100%', background: '#ECECEC' },
+  media: { height: 160, width: 160 }
+}));
 
-const ImageStyle = styled.img`
-  width: 160px;
-  height: 160px;
-  border-radius: 4px;
-`;
-
-const NoMarginText = styled.p`
-  margin: 0;
-`;
-
-const ItalicText = styled.p`
-  font-style: italic;
-  margin: 0;
-`;
-
-const NoMarginHeading = styled.h2`
-  margin: 0;
-`;
-
-const GameInfoStyle = styled.div`
-  flex: 1;
-  display: flex;
-  gap: 8px;
-  flex-direction: column;
-`;
-
-const RatingContainer = styled.div`
-  display: flex;
-  align-items: center;
-  height: fit-content;
-  gap: 8px;
-`;
+const DEFAULT_IMG =
+  'https://socialistmodernism.com/wp-content/uploads/2017/07/placeholder-image.png';
 
 const GameItem = ({ data }: { data: GameItemType }) => {
+  const classes = useStyles();
+  const getImageSrc = () => (data.pictures === undefined ? DEFAULT_IMG : data.pictures[0]);
```

Comment:
On line 15, probably doing check using `data.pictures ? DEFAULT_IMG : data.pictures[0]` might be better, just in case it's empty array?